local_dev_files/build_dev_env.sh

```diff
@@ -86,7 +86,8 @@ if [ -z "$MODULE_NAME" ]; then
     --queue-include-dir $basedir/env/cloud/rds \
     --queue-include-dir $basedir/env/cloud/dynamodb \
     --queue-include-dir $basedir/env/cloud/lambdas \
-    --queue-include-dir $basedir/env/cloud/vpn
+    --queue-include-dir $basedir/env/cloud/vpn \
+    --queue-include-dir $basedir/env/cloud/glue
```

Comment:
I will let @bryan-robitaille confirm that but I think `glue` was not deployed to our scratch accounts for cost saving reasons. If we want to keep it in there we could make it optional (off by default) using a command option like `make build_env glue=on`

Reply:
You're right I didn't include the glue service as part of the base "scratch dev accounts" for cost savings.  There were many modules that I left out of the "development" suite as they were really unique and would only be used if a specific service was being tweaked.
I'll leave it up to you to decide whether or not to update the make file and subsequent build_env script to check for a specific argument flag.  The other option is to run the module on the command line manually when needed.